### PR TITLE
Updated bower dependecies

### PR DESCRIPTION
**Pull request by botraunak**
and updated breaking UI code

* were unneccsarry after upgrading to latest angular material

### CODE REVIEW

File: app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html [hunk 1]
```diff
@@ -9,25 +9,25 @@
         <md-card-content>
             <form name="addBeneficiaryForm" ng-submit="addBeneficiaryForm.$valid && vm.submit()">
                 <md-input-container class="md-block md-icon-float">
-                    <label>{{ 'label.input.accountType' | translate }} <span class="required">*</span></label>
```
Comment (gauravkuelap) [inline app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html]: @botraunak Are these fields not required ? Why we removed this ?

File: bower.json [hunk 2]
```diff
@@ -5,25 +5,22 @@
...
-    "angular-ui-router": "~0.2.17",
-    "angular-material": "1.0.0",
-    "angular": "1.4.9",
-    "angular-nvd3": "1.0.0-rc.2",
-    "components-font-awesome": "~4.5.0",
-    "angular-resource": "~1.5.0",
-    "angular-material-data-table": "^0.9.12",
+    "jquery": "^3.2.1",
+    "angular": "^1.6.5",
+    "angular-material": "^1.1.4",
```
Comment (gauravkuelap) [inline bower.json]: @botraunak After upgrading the versions, did you tested the application once entirely so we don't have any issues arising due to change in version ?

File: app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html (hunk 1 above)
Comment (botraunak) [reply app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html]: I just removed the asterik. on the latest material the select already has an asterik.

File: bower.json (hunk 2 above)
Comment (gauravsaini03) [inline bower.json]: @botraunak After upgrading the versions, did you tested the application once entirely so we don't have any issues arising due to change in version ?

Comment (botraunak) [reply bower.json]: Yes. santosh has done QA and all on my fork. :)

File: app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html (hunk 1 above)
Comment (gauravsaini03) [reply app/src/beneficiaries/beneficiaries-add/beneficiaries-add.html]: ok, then its fine.

File: bower.json (hunk 2 above)
Comment (gauravsaini03) [reply bower.json]: ok :)

**Pull request merged**